### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by mishakoshilka**
- [DEMO LINK](https://mishakoshilka.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://mishakoshilka.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,36 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="images\logo.png"
+        alt="logotype Moyo"
+        class="logo__image">
```
Comment (volodymyr-soltys97) [inline src/index.html:20]: You need to check and fix it everywhere
image

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  height: 60px;
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:48]: These are default values. No need to write it
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code more perfect

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+  text-decoration: none;
+  height: 60px;
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav__link:hover {
```
Comment (mykhalenych) [inline src/style.css:50]:
```suggestion
.nav__link:hover,
.nav__link.is-active {
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  height: 60px;
+  position: relative;
+  font-family: "Roboto", sans-serif;
```
Comment (mykhalenych) [inline src/style.css:44]: better add fonts styles for all elements in html selector

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**